### PR TITLE
What is killbill improvements

### PR DESCRIPTION
**Pull request by sbrossie**

### CODE REVIEW

**Comment by sbrossie** [comment]
@pierre Build locally for real html page generation

File: userguide/tutorials/what_is_kill_bill.adoc [hunk 1]
```diff
@@ -8,53 +8,111 @@ The *subscription billing* features let you create recurring or usage based subs
...
-https://www.youtube.com/watch?v=f2IHcz3OLYo[image:http://img.youtube.com/vi/f2IHcz3OLYo/0.jpg[align=center]]
 
-== The Kill Bill Platform
+So, in the diagram above, we can see that:
+
+* Customers interact with your website, which then issues REST API calls to Kill Bill. **You are responsible for designing and building that piece**.
+* Kill Bill, with its database, **manages all the state associated with the subscriptions, invoices, payments, etc.**
+* KAUI is the **Back-office UI for your teams**, and it includes role based access control to provide different priviledges for different teams/users:
+  ** Your customer support team will use to access/modify consumer accounts data
+  ** Your product team  will use to configure catalog information, billing rules, ..
```
Comment (pierre) [inline userguide/tutorials/what_is_kill_bill.adoc]: `...` (missing third dot)

File: userguide/tutorials/what_is_kill_bill.adoc [hunk 2]
```diff
@@ -8,53 +8,111 @@ The *subscription billing* features let you create recurring or usage based subs
...
+
+
+=== Deployment
+
+Kill Bill is packaged as a WAR and is therefore executed within a web container (typically Tomcat). While you can deploy Kill Bill directly as a WAR inside a container, we also provide easier deployment options:
+
+// TODO link/doc for all these sections
+* Docker containers 
+* 1-click button AWS instances
+* NEW: Hosted cloud deployment
```
Comment (pierre) [inline userguide/tutorials/what_is_kill_bill.adoc]: I would remove the `NEW`

File: userguide/tutorials/what_is_kill_bill.adoc [hunk 3]
```diff
@@ -8,53 +8,111 @@ The *subscription billing* features let you create recurring or usage based subs
...
+
+Most billing systems out there have been designed to be batch oriented, i.e invoice generation and payments are made in batches at specific points in time. We have taken a different approach, and have built a system that is **event driven**, and where operations will happen in sequence.
+
+A typical scenario is one where a subscription gets created, eventually leading to the creation of an _invoice_ and a _payment_. Reusing our example from above, i.e a deployment with the 2 plugins, the following would occur:
+
+{empty} +
+
+[caption=""] 
+//.Kill Bill: Big Picture
+image::../assets/what_is_killbill/KillBill_Subscription_Flow.png[format=png,align=center,title-align=center]
```
Comment (pierre) [inline userguide/tutorials/what_is_kill_bill.adoc:76]: There's a bunch of white space at the bottom of the flow diagram, could we remove it?

**Review by pierre — APPROVED** [review]
👍

**Pull request merged**